dlp/snippets/redact/redact.go

```diff
@@ -81,11 +81,11 @@ func redactImage(w io.Writer, projectID string, infoTypeNames []string, bytesTyp
 	// Send the request.
 	resp, err := client.RedactImage(ctx, req)
 	if err != nil {
-		return fmt.Errorf("RedactImage: %w", err)
+		return fmt.Errorf("RedactImage: %v", err)
 	}
 	// Write the output file.
 	if err := ioutil.WriteFile(outputPath, resp.GetRedactedImage(), 0644); err != nil {
-		return fmt.Errorf("ioutil.WriteFile: %w", err)
+		return fmt.Errorf("ioutil.WriteFile: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.